src/test/java/com/iexec/common/utils/BytesUtilsTests.java

```diff
@@ -61,6 +61,50 @@ public void shoudNotBeHexaStringSinceEmpty() {
         assertFalse(isHexaString(""));
     }
 
+    // isHexStringWithProperByteSize(..) tests
+
+    @Test
+    void shouldBeHexStringWithProperByteSize() {
+        assertTrue(isHexStringWithProperByteSize("0xabc123", 6));
+    }
+
+    @Test
+    void shouldNotBeHexStringWithProperByteSizeSinceExpectedSizeTooSmall() {
+        assertFalse(isHexStringWithProperByteSize("0xabc123", 0));
+    }
+
+    @Test
+    void shouldNotBeHexStringWithProperByteSizeSinceEmptyInput() {
+        assertFalse(isHexStringWithProperByteSize("", 6));
+    }
+
+    @Test
+    void shouldNotBeHexStringWithProperByteSizeSinceNoPrefix() {
+        assertFalse(isHexStringWithProperByteSize("abc123", 6));
+    }
+
+    @Test
+    void shouldNotBeHexStringWithProperByteSizeSinceNotLowerCase() {
```

Comment:
Why is that? In this case shouldn't the method be named `isLowercaseHexStringWithProperByteSize()`

Reply:
This is what I was updating, let me send you a next version